test/integration_test.rb

```diff
@@ -0,0 +1,116 @@
+require_relative 'test_helper'
+
+class IntegrationTest < Minitest::Test
+  def setup
+    build_app
+  end
+
+  def teardown
+    FakeWeb.clean_registry
+    FakeWeb.last_request = nil
+  end
+
+  def test_authorize
+    response = authorize('snowdevil.myshopify.com')
+    assert_equal 302, response.status
+    assert_match /\A#{Regexp.quote("https://snowdevil.myshopify.com/admin/oauth/authorize?")}/, response.location
```

Comment:
[according to this](http://api.rubyonrails.org/classes/ActionDispatch/Assertions/ResponseAssertions.html) you could do:
```
assert_redirected_to /\A#{Regexp.quote("https://snowdevil.myshopify.com/admin/oauth/authorize?")}/
```